modules/reviews/review-request.php

```diff
@@ -0,0 +1,457 @@
+						dataType: "json",
+						url: '<?php echo self::$api_url; ?>',
+						data: {
+							trigger_group: trigger.group,
+							trigger_code: trigger.code,
+							reason: reason,
+							uuid: '<?php echo $uuid; ?>'
+						}
+					});
+					<?php endif; ?>
+				}
+
+				$(document)
+					.on('click', '.ppch-notice .ppch-dismiss', function (event) {
+						var $this = $(this),
+							reason = $this.data('reason'),
+							notice = $this.parents('.ppch-notice');
+
+						notice.fadeTo(100, 0, function () {
+							notice.slideUp(100, function () {
+								notice.remove();
+							});
+						});
+
+						dismiss(reason);
+					})
+					.ready(function () {
+						setTimeout(function () {
+							$('.ppch-notice button.notice-dismiss').click(function (event) {
+								dismiss('maybe_later');
+							});
+						}, 1000);
+					});
+			}(jQuery));
+		</script>
+
+		<div class="notice notice-success is-dismissible ppch-notice">
+
+			<p>
+				<?php echo $tigger['message']; ?>
+			</p>
+			<p>
+                <a class="button button-primary ppch-dismiss" target="_blank" href="https://wordpress.org/support/plugin/publishpress-checklists/reviews/?rate=5#rate-response" data-reason="am_now">
+                    <strong><?php _e( 'Ok, you deserve it', 'publishpress-checklists' ); ?></strong>
+                </a> <a href="#" class="button ppch-dismiss" data-reason="maybe_later">
+                    <?php _e( 'Nope, maybe later', 'publishpress-checklists' ); ?>
+                </a> <a href="#" class="button ppch-dismiss" data-reason="already_did">
+                    <?php _e( 'I already did', 'publishpress-checklists' ); ?>
+                </a>
+			</p>
+
+		</div>
+
+		<?php
+	}
+
+	/**
+	 * Checks if notices should be shown.
+	 *
+	 * @return bool
+	 */
+	public static function hide_notices() {
+		$conditions = array(
+			self::already_did(),
+			self::last_dismissed() && strtotime( self::last_dismissed() . ' +2 weeks' ) > time(),
+			empty( self::get_trigger_code() ),
+		);
+
+		return in_array( true, $conditions );
+	}
+
+	/**
+	 * Gets the last dismissed date.
+	 *
+	 * @return false|string
+	 */
+	public static function last_dismissed() {
+		$user_id = get_current_user_id();
+
+		return get_user_meta( $user_id, '_ppch_reviews_last_dismissed', true );
+	}
+
+	/**
+	 * Sort array by priority value
+	 *
+	 * @param $a
+	 * @param $b
+	 *
+	 * @return int
+	 */
+	public static function sort_by_priority( $a, $b ) {
+		if ( ! isset( $a['pri'] ) || ! isset( $b['pri'] ) || $a['pri'] === $b['pri'] ) {
+			return 0;
+		}
+
+		return ( $a['pri'] < $b['pri'] ) ? - 1 : 1;
+	}
+
+	/**
+	 * Sort array in reverse by priority value
+	 *
+	 * @param $a
+	 * @param $b
+	 *
+	 * @return int
+	 */
+	public static function rsort_by_priority( $a, $b ) {
+		if ( ! isset( $a['pri'] ) || ! isset( $b['pri'] ) || $a['pri'] === $b['pri'] ) {
+			return 0;
+		}
+
+		return ( $a['pri'] < $b['pri'] ) ? 1 : - 1;
+	}
+
+}
+
+Ppch_Modules_Reviews::init();
```

Comment:
This call to the method `init` is not required if the module is registered on the method `LegacyPlugin::getModulesBasePath`.